src/util/dataset.js

```
@@ -0,0 +1,75 @@
+ * BUT WITHOUT ANY WARRANTY, WITHOUT EVEN THE IMPLIED WARRANTY OF MERCHANTABILITY OR FITNESS FOR
+ * A PARTICULAR PURPOSE. SEE THE APPLICABLE LICENSES FOR MORE DETAILS.
+ *
+ */
+import { isObject, isArray } from './type';
+import Theme from '../feature/token';
+import { CHART_TYPE } from './constants';
+
+const DEFAULT_KEY = 'product';
+
+const getDatasetData = option => {
+  if (option.dataset && option.dataset.source) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)